ksp_plugin_adapter/flight_planner.cs

```diff
@@ -70,20 +70,50 @@ class FlightPlanner : VesselSupervisedWindowRenderer {
       return;
     }
 
+    using (new UnityEngine.GUILayout.HorizontalScope()) {
+      int flight_plans = plugin.FlightPlanCount(vessel_guid);
+      int selected_flight_plan = plugin.FlightPlanSelected(vessel_guid);
+      for (int i = 0; i < flight_plans; ++i) {
+        var id = new string(L10N.CacheFormat("#Principia_AlphabeticList")[i],
+                            1);
+        if (UnityEngine.GUILayout.Toggle(i == selected_flight_plan, id,
+                                         "Button",
+                                         GUILayoutWidth(1)) &&
+            i != selected_flight_plan) {
+          plugin.FlightPlanSelect(vessel_guid, i);
+          final_time_.value_if_different =
+              plugin.FlightPlanGetDesiredFinalTime(vessel_guid);
+          ClearBurnEditors();
+          UpdateVesselAndBurnEditors();
+        }
+      }
+      if (flight_plans < max_flight_plans &&
+          UnityEngine.GUILayout.Button("+", GUILayoutWidth(1))) {
```

Comment:
This `+` is going to look weird when there is no flight plan.  I would suggest to keep a text that says "create" somehow.